docs/configuration.rst

```diff
@@ -357,12 +172,12 @@ The GA4GH reference server is a `Flask application <http://flask.pocoo.org/>`_
 and uses the standard `Flask configuration file mechanisms
 <http://flask.pocoo.org/docs/0.10/config/>`_.
 Many configuration files will be very simple, and will consist of just
-one directive instructing the server where to look for data; for
+one directive instructing the server where to find the data repository;
 example, we might have
 
 .. code-block:: python
 
-    DATA_SOURCE = "/path/to/data/root"
+    DATA_SOURCE = "/path/to/ga4gh-repo.db"
```

Comment:
I think the server needs read and write access to the directory since it writes .shm and .wal files. https://www.sqlite.org/tempfiles.html